packages/object-schema/src/types.ts

```diff
@@ -36,16 +36,19 @@ export interface PropertyDefinition {
 	/**
 	 * The strategy to merge the property.
 	 */
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any -- This is a generic type
```

Comment:
[Docs] Similar note here: this PR might not be the place to clean up types, but I wouldn't want to leave a note suggesting `any`s are the right approach.

```suggestion	
	// eslint-disable-next-line @typescript-eslint/no-explicit-any -- https://github.com/eslint/rewrite/pull/90/files#r1687133551
```